package/Dockerfile

```diff
@@ -16,23 +20,48 @@ RUN zypper -n install cmake kmod curl nfs-client nfs4-acl-tools fuse git gcc \
 RUN wget https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.3.2/grpc_health_probe-linux-${ARCH} -O /usr/local/bin/grpc_health_probe && \
     chmod +x /usr/local/bin/grpc_health_probe
 
+# Build gperftools
+# 1. Just use tcmalloc lib from gperftools installation
+# 2. tcmalloc also supports arm64 platform, but we only focus on amd64 first currently.
+#    arm64 will be supported after more testing.
+RUN if [ ${ARCH} == "amd64" ]; then \
+        cd /usr/src && \
+        git clone https://github.com/gperftools/gperftools.git&& \
+        cd gperftools && \
+        git checkout db6d94d9893fdd8c74028b5113cd9185be1e3111 && \
+        ./autogen.sh && \
+        ./configure && \
+        make && \
+        make install; \
+        zypper -n remove autoconf libtool libunwind-devel gcc-c++; \
+        rm -rf /usr/src/gperftools; \
+    fi
+
+RUN ldconfig
```

Comment:
Suggest moving ldconfig to the end of the previous RUN to avoid an extra layer in the final image.